{
	"about": "https://jvndb.jvn.jp/ja/contents/2015/JVNDB-2015-008263.html",
	"title": "python-dbusmock における入力確認に関する脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2015/JVNDB-2015-008263.html",
	"description": "python-dbusmock には、入力確認に関する脆弱性が存在します。",
	"identifier": "JVNDB-2015-008263",
	"references": [
		{
			"text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-1326",
			"source": "CVE",
			"id": "CVE-2015-1326"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2015-1326",
			"source": "NVD",
			"id": "CVE-2015-1326"
		},
		{
			"text": "https://jvndb.jvn.jp/ja/cwe/CWE-20.html",
			"id": "CWE-20",
			"title": "不適切な入力確認(CWE-20)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:python-dbusmock_project:python-dbusmock",
			"version": "2.2",
			"vendor": "python-dbusmock project",
			"product": "python-dbusmock"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "9.3",
			"type": "Base",
			"severity": "High",
			"vector": "AV:N/AC:M/Au:N/C:C/I:C/A:C"
		},
		{
			"version": "3.0",
			"score": "8.8",
			"type": "Base",
			"severity": "High",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H"
		}
	],
	"date": "2019-05-30T17:32+09:00",
	"issued": "2019-05-30T17:32+09:00",
	"modified": "2019-05-30T17:32+09:00"
}
